doc/api/errors.md

```diff
@@ -450,18 +452,35 @@ added properties.
 
 #### error.code
 
-Returns a string representing the error code, which is always `E` followed by
-a sequence of capital letters, and may be referenced in `man 2 intro`.
+The `error.code` property is a string representing the error code, which is always
+`E` followed by a sequence of capital letters, and may be referenced in `man 2 intro`.
 
 #### error.errno
 
-Returns a number corresponding to the **negated** error code, which may be
-referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
-`-2` because the error code for `ENOENT` is `2`.
+The `error.errno` property is a number or a string.
+The number corresponds to the **negated** error code, which may be referenced in
```

Comment:
I'm not even sure I'd mention the 'negated' part because the description on the libuv page makes it sound as if the Windows error codes are not negated. If we need to describe the error code, perhaps we should just say 'positive error code'?